docs/getting-started/intro.md

```diff
@@ -14,7 +14,7 @@ Everything you need to know about getting started with Rancher Turtles.
-| Cluster API Operator     | `>=v0.5.1`                               | Using [helm chart](https://github.com/kubernetes-sigs/cluster-api-operator/blob/main/docs/README.md#method-2-use-helm-charts) install or as a [dependency](./install_turtles_operator.md#install-cluster-api-operator-as-a-helm-dependency) (recommended) to the `Rancher Turtles`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| Cert-manager             | `>=v1.12.0`                              | Using [helm](https://cert-manager.io/docs/installation/helm/#installing-with-helm) based installation or via [kubectl apply](https://cert-manager.io/docs/installation/#default-static-install) command or as a helm [dependency](./install-rancher-turtles/using_rancher_dashboard.md) (recommended)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| Cluster API Operator     | `>=v0.5.1`                               | Using [helm chart](https://github.com/kubernetes-sigs/cluster-api-operator/blob/main/docs/README.md#method-2-use-helm-charts) and then [manually installing Turtles](./install-rancher-turtles/using_helm.md) or letting Turtles take care of it as a [dependency](./install-rancher-turtles/using_rancher_dashboard.md) (recommended) to the `Rancher Turtles`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
```

Comment:
I'd probably switch these so the recommended option is first (i.e. via the turtles chart) and the helm method second (and maybe marked as advanced use cases)

Reply:
@richardcase did you mean: https://github.com/rancher/turtles-docs/pull/75/commits/e5feb2ae971d0bb014226a44f4ecf31b74c2490f?